ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -14,8 +14,18 @@ def is_tensor(obj):
 # def is_storage(obj):
 # 	return ivy.is_storage(obj)
 
-# def is_complex(obj):
-# 	return ivy.is_complex(obj)
+
+def is_complex(obj):
```

Comment:
I believe the test relevant test is failing for this PR, I think it is due to the fact that you have labelled the parameter as `obj` and not `input` which is consistent with the PyTorch documentation :)